### ISSUE TITLE
[Enhancement]: Accessibility: Add scope to products attributes table headers

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43608]
### Describe the solution you'd like

The product attributes table on the single product is coded with row headers but does not have the scope added to the th tag:
https://github.com/woocommerce/woocommerce/blob/758df4854d7b627fc26eb5c0860dc492dea6a06e/plugins/woocommerce/templates/single-product/product-attributes.php#L29

In VoiceOver, the heading is not clearly announced in the table, particularly if there is only a single row. Adding scope="row" to the TH tags to connect it more explicitly to the TD cell would likely resolve this.

### Describe alternatives you've considered

N/A

**Comment by dinhtungdu** [comment]
> Adding scope="row" to the TH tags to connect it more explicitly to the TD cell would likely resolve this.

Sounds like the way to go for me. Thank you so much Amber! I added this to our backlog to work on it later.

**Comment by amesplant** [comment]
I am looking into this issue this week

**Comment by amesplant** [comment]
@dkotter sending this to you for code review
